src/general/OptionsMenu.cs

```diff
@@ -853,6 +859,9 @@ public void ApplySettingsToControls(Settings settings)
         bloomEffectToggle.ButtonPressed = settings.BloomEnabled;
         bloomSlider.Value = settings.BloomStrength;
         bloomSlider.Editable = settings.BloomEnabled || !DisableInactiveSliders;
+        blurEffectToggle.ButtonPressed = settings.MicrobeBackgroundBlurEnabled;
+        blurSlider.Value = settings.MicrobeBackgroundBlurStrength;
+        blurSlider.Editable = settings.MicrobeBackgroundBlurEnabled || !DisableInactiveSliders;
```

Comment:
Disabled sliders cause issues with the focus flow (as we currently have no visual way disabled sliders show they have focus), so based on my previous comment I kind of recommend we'd just have a single blur strength slider that when dragged to 0 acts as the disabled state.